src/test/modelmanager_test.cpp

```diff
@@ -1382,11 +1383,7 @@ TEST_F(ModelManager, HandlingInvalidLastVersion) {
     status = manager.getModelInstance(modelDirectory.name, 3, modelInstance3, modelInstanceUnloadGuard);
     modelInstanceUnloadGuard.reset();
     ASSERT_EQ(status, ovms::StatusCode::MODEL_VERSION_NOT_LOADED_YET);
-
-#ifdef _WIN32
-    // FIXME: TODO: Manual unload required because OVCORE keeps the file handle opened after modelLoad, preventing the test to remove the directory.
     modelInstance2->unloadModelComponents();
```

Comment:
unloadModelComponents need to be deleted also - this was not part of the test oryginally.

Reply:
done